### PR TITLE
More Survivors

### PR DESCRIPTION
**Pull request by Tunguso4ka**
## About the PR

Adds more gearPresets for Colonists. Thats all.

### CODE REVIEW

**Comment by QuietlyWhisper** [comment]
This might cause issues with the database for touching playtime trackers, will wait for smug to confirm

**Comment by Tunguso4ka** [comment]
> This might cause issues with the database for touching playtime trackers, will wait for smug to confirm

I'l change Survivor playtime back then.

File: Resources/Prototypes/_RMC14/Roles/Jobs/Survivor/colonist.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ﻿- type: job
   parent: CMJobBase
-  id: CMSurvivor
-  name: cm-job-name-survivor
```
Comment (DrSmugleaf) [inline Resources/Prototypes/_RMC14/Roles/Jobs/Survivor/colonist.yml:4]: Are they actually called colonists in 13? I can't find them on my local server so I can't check, but from the code and wiki they seem to be called survivors

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_RMC14/Roles/Jobs/Survivor/colonist.yml (hunk 1 above)
Comment (Tunguso4ka) [reply Resources/Prototypes/_RMC14/Roles/Jobs/Survivor/colonist.yml:4]: oh theres... many of them
```
#define JOB_COLONIST "Colonist"
#define JOB_PASSENGER "Passenger"
#define JOB_SURVIVOR "Survivor"
#define JOB_SYNTH_SURVIVOR "Synth Survivor"
#define JOB_CO_SURVIVOR "CO Survivor
```

Comment (Tunguso4ka) [reply Resources/Prototypes/_RMC14/Roles/Jobs/Survivor/colonist.yml:4]: gonna role back to survivor then.

**Review by QuietlyWhisper — APPROVED** [review]

**Pull request merged**